types.d.ts

```diff
@@ -0,0 +1,55 @@
+export type MontiEvent = any
+export type KadiraInfo = any
+
+export type ConnectOptions = {
+    enableErrorTracking?: boolean
+    endpoint?: string
+    hostname?: string
+    uploadSourceMaps?: boolean
+    recordIPAddress?: string
+    eventStackTrace?: boolean
+    disableNtp?: boolean
+    stalledTimeout?: number
+}
+
+export type TrackErrorOptions = {
+    type?: string
```

Comment:
While any string is allowed, the UI only supports certain values. It probably would be good to list them here: `method`, `client`, `sub`, `server-crash`, `server-internal`